ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -123,3 +123,7 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def collapse_repeated(labels, seq_length, name=None):
```

Comment:
I don't think there's a `collapse_repeated` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `collapse_repeated` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.collapse_repeated` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)